lasagne/updates.py

```diff
@@ -132,8 +138,9 @@ def sgd(loss_or_grads, params, learning_rate):
 
 
 def apply_momentum(updates, params=None, momentum=0.9):
-    """Returns a modified update dictionary that includes momentum terms.
+    """Applies momentum
```

Comment:
This is just copying the function name now -- was the previous version too long for the table? In that case, we should probably aim for something in between. I'll check how this PR renders in Sphinx and try to come up with something.